src/data/invalid/Database-functional_annotation_agg.yaml

```diff
@@ -1,4 +1,5 @@
 #invalid b/c a required slot is missing
 functional_annotation_agg:
-  - metagenome_annotation_id: nmdc:8253bcdcd0387177ff895c38a047c719
+  - metagenome_annotation_id: nmdc:wfmgan-3a9-A.3_43_33
```

Comment:
This should not be a valid metagenome_annotation_id, it should be a WorkflowExeuction identifier, not a gene id There is either an issue with a previous backmerge or when this was fixed in nmdc-schema hasn't been merged yet.  Please update.

Reply:
I've made a ticket to check on the pattern constraint issue for myself https://github.com/microbiomedata/nmdc-schema/issues/2172, fixing that is outside the scope of this PR.

Reply:
I'm going to make a separate issue to correct this ID after the backmerge is done and the pattern constraint is in, so this PR isn't held up